index.html

```diff
@@ -5,110 +5,127 @@
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&display=swap" rel="stylesheet">
-    <link href="normalize.css" rel="stylesheet">
+    <link href="css/normalize.css" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
 </head>
 
 <body class="page-body">
+    <div class="container">
+        <header class="main-header ">
+            <nav class="main-nav">
+                <a class="main-nav-logo" href="">
+                    <img src="img/logo.svg" width="138" height="70" alt="city of SEDONA">
+                </a>
+                <ul class="main-nav-list list-reset">
+                    <li><a class="main-nav-link" href="">информация</a></li>
+                    <li><a class="main-nav-link" href="">фото и видео</a></li>
+                    <li><a class="main-nav-link" href="">карта штата</a></li>
+                    <li><a class="main-nav-link" href="catalog.html">гостиницы</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main class="main">
+            <section class="hero">
+                <img class="hero-welcome" src="img/welcome.svg" width="457" height="351" alt="WELCOME-to the gorgeous- SEDONA">
+            </section>
+            <h1 class="main-title">
+                Седона - небольшой городок в Аризоне, заслуживающий большего!
+            </h1>
+            <section class="features">
+                <h2 class="reasons-why">
```

Comment:
```suggestion
                <h2 class="features-text">
```

или features-description